mk64.ld

```diff
@@ -656,13 +656,11 @@ SECTIONS
       #else
       BUILD_DIR/data/sound_data/sequences.o(.data);
       #endif
+      . = ALIGN(0x40);
    }
    END_SEG(sequences)
-   BEGIN_SEG(instrument_sets, 0x282E80)
+   BEGIN_SEG(instrument_sets, .)
```

Comment:
Can you compare with the rest of the file and see if the `.` is sensical?

Reply:
That segment is shifted around between US and EU, that change was made after comparing them using their respective map files